react-toggle-switch-component/my-app/src/ToggleButton.jsx

```diff
@@ -0,0 +1,23 @@
+import './ToggleButton.css';
+import { useState } from 'react';
+
+export default function ToggleButton() {
+  const [isOn, setIsOn] = useState(false);
+  let text = '';
+  if (isOn) {
+    text = 'On';
+  } else {
+    text = 'Off';
+  }
+  function handleClick() {
+    setIsOn(!isOn);
+  }
+  return (
+    <div className="container">
+      <div className={`button-background-${text}`} onClick={handleClick}>
```

Comment:
This is clever to use the text in your CSS, but it doesn't localize to different languages, so it's not something you would do in "real life".